### PR TITLE
Next gen node snippets taskrouter

### PR DESCRIPTION
**Pull request by hortega**
- [x] Activities
- [x] Events
- [x] Jwts
- [x] Reservations
- [x] Statistics
- [x] Taskchannels
- [x] Taskqueues
- [x] Tasks
- [x] Twiml
- [x] Worker-channels
- [x] Worker-reservations
- [x] Workers
- [x] Workflows
- [x] Workspaces

### CODE REVIEW

**Comment by hortega** [comment]
@joliveros could you peer review this pull request? thx!

**Comment by juancarlospaco** [comment]
Looks good for me. :+1:

File: rest/taskrouter/activities/instance/get/example-1/example-1.2.x.js [hunk 1]
```diff
@@ -2,13 +2,13 @@
 // These vars are your accountSid and authToken from twilio.com/user/account
 var twilio = require('twilio');
 
-var accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
-var authToken = "your_auth_token";
-var workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
-var activitySid = "WAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var authToken = 'your_auth_token';
+var workspaceSid = 'WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
```
Comment (joliveros) [inline rest/taskrouter/activities/instance/get/example-1/example-1.2.x.js]: can you restore this so it doesn't get published again?

File: rest/taskrouter/activities/list/get/example-1/example-1.3.x.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+// These consts are your accountSid and authToken from https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
+const workspaceSid = 'WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const client = require('twilio')(accountSid, authToken);
+
+client.taskrouter.v1
+  .workspaces(workspaceSid)
+  .activities
+  .list()
```
Comment (joliveros) [inline rest/taskrouter/activities/list/get/example-1/example-1.3.x.js]: what do you think about using the #each method provided in the sdk? https://github.com/TwilioDevEd/api-snippets/blob/next-gen-node-snippets-rest-part-1/rest/addresses/list-get-example-2/list-get-example-2.3.x.js#L9

File: rest/taskrouter/statistics/workflow/example-1/example-1.3.x.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+const client = require('twilio')(accountSid, authToken);
+
+client.taskrouter.v1
+  .workspaces(workspaceSid)
+  .workflows(workflowSid)
+  .statistics()
+  .fetch()
+  .then((responseData) => {
+      console.log(responseData.cumulative.avg_task_acceptance_time);
+      console.log(responseData.cumulative.tasks_entered);
```
Comment (joliveros) [inline rest/taskrouter/statistics/workflow/example-1/example-1.3.x.js]: double indent

File: rest/taskrouter/tasks/list/get/example-2/example-2.3.x.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+// These consts are your accountSid and authToken from https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
+const workspaceSid = 'WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const client = require('twilio')(accountSid, authToken);
+
+client.taskrouter.v1
+  .workspaces(workspaceSid)
+  .tasks
+  .list({'TaskQueueSid': 'WQf855e98ad280d0a0a325628e24ca9627'})
```
Comment (joliveros) [inline rest/taskrouter/tasks/list/get/example-2/example-2.3.x.js]: you could drop the quotes here

File: rest/taskrouter/twiml/example3/example/example.3.x.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+app.post('/enqueue_call', function(request, response) {
+    const resp = new twilio.TwimlResponse();
+
+    const arr = {account_number: '12345abcdef'};
+    const json = JSON.stringify(arr);
+
+    resp.enqueue(
+      {workflowSid: 'WW0123456789abcdef0123456789abcdef'},
+      (node) => {
+        node.task(json, {
```
Comment (joliveros) [inline rest/taskrouter/twiml/example3/example/example.3.x.js]: Just my opinion, but you could move this anonymous function and define it above for readability. I can read it either way though. +1 for readability throughout!

File: rest/taskrouter/workflows/list/post/example-1/example-1.3.x.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+client.taskrouter.v1
+  .workspaces(workspaceSid)
+  .workflows
+  .create({
+    friendlyName: 'Sales, Marketing, Support Workflow',
+    assignmentCallbackUrl: 'http://example.com',
+    fallbackAssignmentCallbackUrl: 'http://example2.com',
+    taskReservationTimeout: '30',
+    configuration: JSON.stringify(configuration)
```
Comment (joliveros) [inline rest/taskrouter/workflows/list/post/example-1/example-1.3.x.js]: you figured it out! nice +1

**Comment by joliveros** [comment]
Looks good @hortega, super tidy. Just a few comments let me know what you think about using #each vs #list

File: rest/taskrouter/activities/list/get/example-1/example-1.3.x.js (hunk 2 above)
Comment (hortega) [inline rest/taskrouter/activities/list/get/example-1/example-1.3.x.js]: cool! I missed that method altogether

**Comment by hortega** [comment]
I just applied all the changes you suggested @joliveros , please let me know if I missed anything

**Comment by juancarlospaco** [comment]
Looks good for me. :+1:

**Pull request merged**